cmd/nerdctl/compose/compose_pause.go

```diff
@@ -65,7 +67,7 @@ func composePauseAction(cmd *cobra.Command, args []string) error {
 	for _, c := range containers {
 		c := c
 		eg.Go(func() error {
-			if err := pauseContainer(ctx, client, c.ID()); err != nil {
+			if err := utils.PauseContainer(ctx, client, c.ID()); err != nil {
```

Comment:
I think `PauseContainer` should be put in `pkg/containerutil` since it doesn't depend on `cmd` and it's more related to container.

I would suggest within `containerutil` we can create a new `lifecycle.go` to include all high-level operations around a container/containers (such as `Pause`, `Start`, `Stop`, etc). (We don't necessarily check and move all altogether now :) but gradually we should if that sounds good)

https://github.com/containerd/nerdctl/tree/main/pkg/containerutil